xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/importer/import.css

```diff
@@ -156,15 +156,15 @@
   padding: 3px !important;
 }
 
-#import ul.package li.xitem div.xitemcontainer span.checkbox,
-#import ul.package li.xitem div.xitemcontainer input.space {
+#import ul.package li.xitem button.xitemcontainer span.checkbox,
+#import ul.package li.xitem button.xitemcontainer input.space {
```

Comment:
I'm not convinced by the usage of the button element. We use the anchor element in other suggest drop downs (all suggest pickers) or menus (e.g. More Actions menu). It makes a lot of sense to be able to open a search suggest result in a new tab. You can't do this with the button.